.github/workflows/on_push_pr.yaml

```diff
@@ -7,142 +7,22 @@ on:
-      GPG_PRIVATE_KEY_BASE64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }} # base64 encoded
-    steps:
-      - uses: actions/checkout@v3
-      - run: |
-          git tag "$TAG"
-          if [ -z "$GPG_PASSPHRASE" ]; then
-            echo NO_SIGN=true >> $GITHUB_ENV
-          fi
-
-      - name: Build all platforms:arch
-        run: make ci/fake-prerelease
-      - name: Upload artifacts for next job
-        uses: actions/upload-artifact@v3
-        with:
-          name: windows-packages
-          path: dist/nri-*.zip
-
-  test-build-windows:
-    name: Create MSI
-    runs-on: windows-latest
-    needs: [test-build-nix]
-    env:
-      GOPATH: ${{ github.workspace }}
-      PFX_CERTIFICATE_BASE64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }} # base64 encoded
-      PFX_CERTIFICATE_DESCRIPTION: 'New Relic'
-      PFX_PASSPHRASE:  ${{ secrets.OHAI_PFX_PASSPHRASE }}
-    defaults:
-      run:
-        working-directory: src/github.com/${{ env.ORIGINAL_REPO_NAME }}
-    strategy:
-      matrix:
-        goarch: [amd64,386]
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          path: src/github.com/${{ env.ORIGINAL_REPO_NAME }}
-      - shell: bash
-        run: git tag "$TAG"
-
-      - name: Download artifact from previous job
-        uses: actions/download-artifact@v3
-        with:
-          name: windows-packages
-          path: src/github.com/${{ env.ORIGINAL_REPO_NAME }}/dist/
-
-      - name: Get PFX certificate from GH secrets
-        shell: bash
-        run: |
-          if [ -z "$PFX_CERTIFICATE_BASE64" ]; then
-            echo NO_SIGN=true >> $GITHUB_ENV
-          else
-            printf "%s" "$PFX_CERTIFICATE_BASE64" | base64 -d - > wincert.pfx
-          fi
-
-      - name: Extract .exe
-        shell: pwsh
-        run: build\windows\extract_exe.ps1 "$env:INTEGRATION" ${{ matrix.goarch }} "$env:TAG"
-      - name: Create MSI
-        shell: pwsh
-        run: build\windows\package_msi.ps1 -integration "$env:INTEGRATION" -arch ${{ matrix.goarch }} -tag "$env:TAG" -pfx_passphrase "$env:PFX_PASSPHRASE" -pfx_certificate_description "$env:PFX_CERTIFICATE_DESCRIPTION"
+  push-pr:
+    uses: newrelic/coreint-automation/.github/workflows/reusable_push_pr.yaml@msanmiquel/reusable-push-pipeline
+    secrets:
+      docker_hub_id: ${{ secrets.DOCKER_HUB_ID }}
+      docker_hub_password: ${{ secrets.DOCKER_HUB_PASSWORD }}
+      pfx_passphrase: ${{ secrets.OHAI_PFX_PASSPHRASE }}
+      pfx_certificate_base64: ${{ secrets.OHAI_PFX_CERTIFICATE_BASE64 }}
+      gpg_passphrase: ${{ secrets.OHAI_GPG_PASSPHRASE }}
+      gpg_private_key_base64: ${{ secrets.OHAI_GPG_PRIVATE_KEY_BASE64 }}
+    with:
+      run_test_build_nix: false
+      run_test_build_nix_fake_prerelease: true
+      run_test_build_windows: true
+      tag: "v0.0.0"
+      integration: "apache"
+      original_repo_name: "newrelic/nri-apache"
+      repo_full_name: ${{ github.event.repository.full_name }}
```

Comment:
it is clear we were doing a mess 😓 
it would be nice to remove some of those, but it is not a big deal to keep them

Reply:
Yes. I updated it.